usr/sbin/p11sak/p11sak.c

```diff
@@ -432,6 +432,9 @@ static void print_gen_help(void)
     printf(
             "      --attr [[pub_attrs]:[priv_attrs]] \n");
     printf( "             for asymmetric keys: set individual key attributes, values see above\n");
+    printf( "      -c, --custom-attr [[pub_attrs]:[priv_attrs]] \n");
+    printf( "             for asymmetric keys: set individual key attributes, for more info type:\n");
+    printf( "             man p11sak\n");
```

Comment:
....for more info, see the p11sak man page.

Reply:
But please as `see man 1 p11sak`

Reply:
Changed to see man 1 p11sak